src/main/kotlin/com/openlattice/chronicle/study/StudyUpdate.kt

```diff
@@ -0,0 +1,32 @@
+package com.openlattice.chronicle.study
+
+import com.google.common.base.Preconditions
+import java.time.OffsetDateTime
+
+/**
+ * @author Solomon Tang <solomon@openlattice.com>
+ */
+data class StudyUpdate(
+    val title: String?,
+    val description: String?,
+    val startedAt: OffsetDateTime?,
+    val endedAt: OffsetDateTime?,
+    val lat: Double?,
+    val lon: Double?,
+    val group: String?,
+    val version: String?,
+    val settings: Map<String, Any>?,
+    val contact: String?,
+) {
+    init {
+        Preconditions.checkArgument(
+            title == null || title.isNotEmpty(),
+            "Title cannot be blank."
+        )
+        Preconditions.checkArgument(
+            contact == null || contact.isNotEmpty(),
+            "Contact cannot be blank."
+        )
+    }
+}
```

Comment:
I think it's easier to use the existing Study class here instead of creating a new StudyUpdate class . Also, I noticed that StudyUpdate has a `contact` property but `StudyUpdate` doesn't. Any reason for this?

Reply:
@anzioka using the existing `Study` class forces the caller to populate all the fields for the request. The majority of use cases will be updating a subset of fields, say `title` and `description` and being able to specify only those fields that changed in the request makes it easier on the caller as well as reduces the likelihood of a field being mistakenly updated.

Reply:
the `Study` class has indeed been updated to include the `contact` field. see e666d68

Reply:
Would like to point out Study("title goes here") or { "title" : "Some title" } is all that is needed to create a study. The issue seems to be that there is no way to tell between specified and not specified after construction. I hate all these nullable values and it will be a PITA to maintain if we make changes to Study.